### PR TITLE
New Package: radeontop-0.8

### LINKED ISSUE #1585: Package request: radeontop
https://github.com/clbr/radeontop

### PR DESCRIPTION
**Pull request by logenkain**
closes #1585

### CODE REVIEW

File: srcpkgs/radeontop/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+depends="xf86-video-ati"
+short_desc="View radeon GPU utilization"
+maintainer="Logen K <logen@sudotask.com>"
+license="GPL-3"
+homepage="https://github.com/clbr/radeontop"
+distfiles="https://github.com/clbr/radeontop/archive/${version}.tar.gz"
+checksum=4430e1aaca895d70ea496a9b14010d9d94cadd6269b28a66104af89ff29d8489
+
+do_build() {
+	make CC="$CC" CXX="$CXX" LD="$LD" AR="$AR" RANLIB="$RANLIB" \
```
Comment (xtraeme) [inline srcpkgs/radeontop/template]: Probably just setting `CC` is enough...

File: srcpkgs/radeontop/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'radeontop'
+pkgname=radeontop
+version=v0.8
+revision=1
+wrksrc="${pkgname}-${version#?}"
+build_style=gnu-makefile
+hostmakedepends="pkg-config git gettext"
```
Comment (beefcurtains) [inline srcpkgs/radeontop/template]: Why git? It doesn't appear to depend on it.

**Comment by beefcurtains** [comment]
Please install to ``/usr/bin``

File: srcpkgs/radeontop/template (hunk 2 above)
Comment (xtraeme) [inline srcpkgs/radeontop/template]: And gettext is unnecessary as well.

**Pull request merged**